app/namespaces/data_provider/data_provider_service.py

```diff
@@ -46,7 +46,7 @@ def _get_parsed_record(results):
     for col in results_df.columns:
         # Convert multi select values into an array
         if col in multi_select_cols:
-            multi_select_vals = list(set(getattr(results_df, col).tolist()))
+            multi_select_vals = [e for e in list(set(getattr(results_df, col).tolist())) if e is not None]
```

Comment:
can just say "if e" instead of "if e is not None"

Reply:
Hmm don't think this should ever happen but this suggestion would behave unexpectedly if one of our multi select vals is the integer 0. 

Imo if e is not None is not too unreadable and is more explicit about what we're trying to guard against.

Reply:
sg